### PR TITLE
Fix 3588 aad authentication method policy

### LINKED ISSUE #3588: AADAuthenticationMethodPolicy reference to non existing class instance
### Description of the issue

A reference to the "MSFT_MicrosoftGraphExcludeTarget" instance is made, but that instance doesn't exist.

### Microsoft 365 DSC Version

v1.23.809.1

### Which workloads are affected

Azure Active Directory

### The DSC configuration

_No response_

### Verbose logs showing the problem

_No response_

### Environment Information + PowerShell Version

_No response_

### LINKED ISSUE #3590: IntuneRoleAssignment missing use of Beta Graph module
### Description of the issue

In the resource IntuneRoleAssigment the use of the Beta graph module is missing for these commands:
Get-MgDeviceManagementRoleAssignment
New-MgDeviceManagementRoleAssignment
Update-MgDeviceManagementRoleAssignment
Remove-MgDeviceManagementRoleAssignment

### Microsoft 365 DSC Version

v1.23.809.1

### Which workloads are affected

other

### The DSC configuration

_No response_

### Verbose logs showing the problem

_No response_

### Environment Information + PowerShell Version

_No response_

### PR DESCRIPTION
**Pull request by sandrola**
#### Pull Request (PR) description
Add missing class to AADAuthenticationMethospolicy
Add use of Beta command to IntuneRoleAssignment

#### This Pull Request (PR) fixes the following issues
FIXES #3588 
FIXES #3590

### CODE REVIEW

File: Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof [hunk 1]
```diff
@@ -24,6 +24,12 @@ class MSFT_AADAuthenticationMethodPolicyIncludeTarget
     [Write, Description("The kind of entity targeted. Possible values are: user, group."), ValueMap{"user","group","unknownFutureValue"}, Values{"user","group","unknownFutureValue"}] String TargetType;
 };
 [ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof:27]: I believe the class is already in there as MSFT_AADAuthenticationMethodPolicyExcludeTarget, but the reference is wrong.

**Review by NikCharlebois — CHANGES_REQUESTED** [review]

File: Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof (hunk 1 above)
Comment (sandrola) [reply Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof:27]: Is not really the same. At least the name is different.

Comment (sandrola) [reply Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof:27]: The MSFT_MicrosoftGraphExcludeTarget is used by AuthenticationMethodsRegistrationCampaign and was missing. The other MSFT_AADAuthenticationMethodPolicyExcludeTarget was auto created by the tool.

**Comment by NikCharlebois** [comment]
@sandrola can you fix the merge conflicts? Thanks

**Pull request closed without merging**